### PR TITLE
Move initial whitepaper draft from google docs to Github.

### PR DESCRIPTION
**Pull request by ArthurSens**
A lot of places still need some improvements.

What to look for next:

1. We've been discussing on slack a lot about "Lifecycle".  An observable system goes through design, instrumentation and only then observation. How do we get this concept into the whitepaper?
2. There are a couple of sections unwritten, we need to decide if we're going to remove those sections or not. If we keep them, we need to write :|
3. Some sections were written without proper storytelling, just dumping content. We need to rephrase a lot of paragraphs to make it nice to read.
4. References need to be properly referenced.
5. We need to mention all our amazing contributors that made this whitepaper possible 🤗

### CODE REVIEW

**Comment by bwplotka** [comment]
Thanks a lot! Let's go through this tomorrow?

File: whitepaper.md [hunk 1]
```diff
@@ -0,0 +1,477 @@
+# Observability Whitepaper
+
```
Comment (bwplotka) [inline whitepaper.md:2]: TOC to be added.

File: whitepaper.md [hunk 2]
```diff
@@ -0,0 +1,477 @@
...
+During the design and development of an observable system, it must be instrumented to send or expose telemetry data to a third-party, usually a set of tools, responsible for providing meaningful information out of the exposed data. That telemetry data frequently comes in the form of metrics and logs, long used by software engineering teams, as well as traces, structured events, profiles, and crash dumps. . Each signal has its own purpose and best practices, and their misuse can lead to new problems when running software at scale such as “alert fatigue” and “high cost”.
```
Comment (bwplotka) [inline whitepaper.md:24]:
```suggestion
During the design and development of an observable system, it must be instrumented to send or expose telemetry data to a third-party, usually a set of tools, responsible for providing meaningful information out of the exposed data. That telemetry data frequently comes in the form of metrics and logs, long used by software engineering teams, as well as traces, structured events, profiles, and crash dumps. Each signal has its own purpose and best practices, and their misuse can lead to new problems when running software at scale such as “alert fatigue” and “high cost”.
```

File: whitepaper.md [hunk 3]
```diff
@@ -0,0 +1,477 @@
...
+
+Of course, that’s just the basics. There is whole infrastructure and logic in Prometheus done at the beginning of 2021 to support exemplars on scraping, storing, querying it and even replicating those in a remote write. [Thanos](http://thanos.io/) started to support exemplars, so the Grafana.
+
+It’s also worth mentioning that OpenTelemetry also inherited some form of exemplars from OpenCensus. Those are very similar to OpenMetrics one, just only attachable to histogram buckets. Yet, we are not aware of anyone using or relying on this part of Otel metric protocol anywhere, including relevant implementations like [Go](https://github.com/open-telemetry/opentelemetry-go/issues/559). This means that if you want to have a stable correlation, and an already working ecosystem, OpenMetrics might be the way forward. Plus, [OpenTelemetry slowly adopts OpenMetrics too](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/openmetrics-guidelines.md).
+
+## Use Cases
+
```
Comment (bwplotka) [inline whitepaper.md:335]: Ideas during our call:

* Process of extending observability

Return of investment driven view, iterating etc.

File: whitepaper.md [hunk 4]
```diff
@@ -0,0 +1,477 @@
...
+* Higher-level metric aggregations, downsampling.
+
+Something that is yet to be added is the ability to add exemplars for recording rules and alerts that might aggregate further metrics with exemplars attached. This has been proposed, but the work has yet to be done. Similarly, the downsampling techniques that we discuss for further iterations of Prometheus have to think about downsampling exemplars.
+
+* Native Correlation support in UIs.
+
+Grafana is pioneering in multi-signal links, but many other UIs would use better correlation support given the ways shared in this article. Before the Grafana, the space was pretty fragmented (each signal usually had its own view, rarely thinking about other signals). Prometheus UI is no different. Extra support for linking to other signals or rendering exemplars are to be [added](https://github.com/prometheus/prometheus/issues/8797) there too.
+
+### eBPF
```
Comment (bwplotka) [inline whitepaper.md:445]: Let's create a section for `Contributing to This Whitepaper` where we explain HOW to contribute to those things and let's enumerate those items.

File: whitepaper.md [hunk 5]
```diff
@@ -0,0 +1,477 @@
...
+1. https://www.honeycomb.io/wp-content/uploads/2020/04/observability-maturity-report-4-3-2020-1-1.pdf
+1. https://copyconstruct.medium.com/monitoring-and-observability-8417d1952e1c
+1. https://sre.google/sre-book/service-level-objectives/
+1. https://sre.google/workbook/implementing-slos/
+1. Kalman R. E., "On the General Theory of Control Systems", Proc. 1st Int. Cong. of IFAC, Moscow 1960 1481, Butterworth, London 1961. <https://www.sciencedirect.com/science/article/pii/S1474667017700948?via%3Dihub>
+
+## Colaborators
+
+Extensive list of colaborators of this whitepaper goes here:
+
```
Comment (bwplotka) [inline whitepaper.md:473]: Yes, we need to track it from doc I guess 🙈

**Comment by ArthurSens** [comment]
A couple of things to add:
* Table of contents
* Choosing your signals?
  * “Return On Investment (RoI)” driven assessment
* Use-cases
  * Improving your own system with observability data

For myself:
Move "To be written" to an "Open to contributions section"
Rephrasing could also be "Open to contribution"
Take a look at markdown linking examples on observability-charter

**Review by bwplotka — APPROVED** [review]
Amazing work I think it's time for public comment period, let's merge that in and as agreed I can open PR for public comment period. Is that ok?  

cc @halcyondude @RichiH

**Pull request merged**